content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,362 @@
+   :align: center
+   :alt: Change the Force Removal Strategy for either the Product Categories or Locations.
 
 What happens inside the warehouse?
 ==================================
 
-Imagine a generic warehouse plan, with receiving docks and area, storage locations, picking and
-packing areas, and shipping docks. All products go through all these locations, but some rules,
-such as removal strategies, can have an effect on which products are taken for the pickings.
+Most warehouses share the same important areas: receiving docks and sorting areas, storage
+locations, picking and packing areas, and shipping/loading docks. While all products entering or
+leaving the warehouse might go through each of these locations at some point, removal strategies can
+have an effect on which products are taken, from where, and when.
 
 .. image:: removal/empty-dock.png
    :align: center
-   :alt: empty stock waiting for deliveries at the docks.
+   :alt: Empty stock waiting for deliveries at the docks.
+
+In this example, vendor trucks unload pallets of goods at the docks. Then, operators scan the
+products in the receiving area, with the reception date and expiration date (if applicable). After
+that, products are stored in their respective locations.
+
+In Odoo, receiving products is done either in the *Inventory* or *Barcode* app. In either app, in
+the default kanban view, click :guilabel:`Receipts`, then, click on the individual receipt. Next, in
+the *Inventory* app, click :guilabel:`Edit`, then enter the received quantity in the
+:guilabel:`Done` column. Finally, click :guilabel:`Validate` to receive the products.
 
-Here, vendor trucks unload pallets of goods at the docks. Then, operators scan the products in the
-receiving area, with the receiving date and, if the product has an expiration date, the expiration
-date. After that, products are stored in their respective locations.
+If using the *Barcode* app, scan the product(s), update the quantity, and finally, click
+:guilabel:`Validate`. After products are received in Odoo, the products can be moved to their
+respective storage locations.
 
 .. image:: removal/entering-stocks.png
    :align: center
-   :alt: products entering stock via the receiving area.
+   :alt: Products entering stock via the receiving area.
 
-Next, several orders for the same product are made, but you didn’t receive the goods the same day
-and they don’t have the same expiration date. In that situation, you logically prefer sending those
-with the closest date first. Depending on the removal strategy you chose, Odoo generates a transfer
-with the products fitting your settings the best.
+Next, several orders for the same product are made, but in this example, the goods weren't received
+on the same day, and they don't have the same expiration date. In that situation, logically, sending
+the products with the closest expiration date is preferred. Depending on the chosen removal
```

Comment:
Assuming the products have an expiration date? We haven't really fleshed out the example that far to indicate this., e.g. "Imagine we are receiving cartons of eggs..."